packages/@aws-cdk-testing/framework-integ/test/aws-cognito/test/integ.user-pool-idp.oidc.js.snapshot/integ-user-pool-idp-google.template.json

```diff
@@ -34,9 +34,6 @@
   "poolclient2623294C": {
    "Type": "AWS::Cognito::UserPoolClient",
    "Properties": {
-    "UserPoolId": {
```

Comment:
looks like just order change in this integ test snapshot. We could probably remove this change to make the PR clearer.

Reply:
@GavinZZ It is a little messier but it caught an error with the regex. I'd prefer to leave it in for peace of mind. 🙂

Reply:
Maybe I missed something, but I don't see any integ test changed only snapshots changed. What error are you referring to here?

Reply:
See convo here: https://github.com/aws/aws-cdk/pull/28802#discussion_r1461173898.

Reply:
I'm not following, was that change in provider name a mistaken? I don't see any integration test file (not snapshot) change now.

Reply:
I'll delete the snap changes. I just wanted validation the stack still deployed with the change.